### PR TITLE
feat: what's new modal

### PR DESCRIPTION
**Pull request by shirleyodeng**
## Summary

This PR adds a What's New modal.

Admins can add a new product announcement by changing the params in the `product_announcements:create` rake task and then running it.

Screenshot 2023-04-23 at 10 42 40

Screenshot 2023-04-23 at 10 42 52

## Notion link

https://talentprotocol.notion.site/What-s-new-modal-8df67b9bdbb447ed8d2eaae51cc8f576

### CODE REVIEW

File: app/packs/src/api/product-announcements.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+const markAsRead = (productAnnouncementId) => {
+  const baseHeaders = defaultHeaders();
+  const headers = appendCSRFToken(baseHeaders);
+
+  return axios.put(
+    `/api/v1/product_announcements/${productAnnouncementId}`,
+    { headers: { ...headers } }
+  );
+};
```
Comment (shirleyodeng) [inline app/packs/src/api/product-announcements.js:14]: @RubenSousaDinis - getting the below error when calling this endpoint. Am I sending the incorrect headers / credentials? Or is it just a local issue as I don't have the right credentials?

```
Can't verify CSRF token authenticity.
Completed 500 Internal Server Error in 4ms (Views: 0.2ms | Allocations: 953)

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -28,6 +28,7 @@
           <%= render "shared/navbar" %>
           <%= yield %>
         </div>
+        <%= react_component("ProductAnnouncementModal") %>
```
Comment (shirleyodeng) [inline app/views/layouts/application.html.erb:31]: @RubenSousaDinis is this okay here?

File: lib/tasks/product_announcements.rake [hunk 3]
```diff
@@ -0,0 +1,16 @@
+namespace :product_announcements do
+  desc "Create a product announcement"
+  task create: :environment do
+    params = {
+      title: "New Support Model",
+      content: "We're excited to announce the launch of our latest feature: Career Goals. To get started with it, simply connect to your account and assign a message in your wallet. This will enable you to make the most of our platform yada yada.",
+      link: "https://www.google.com",
+      image_url: "https://fujifilm-x.com/products-cameras-static/gfx100s/assets/images/top/wapk_tonality_img_01.png"
+    }
```
Comment (shirleyodeng) [inline lib/tasks/product_announcements.rake:9]: @RubenSousaDinis - left this as hardcoded params in this rake task. Let me know if you prefer a different way.

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (RubenSousaDinis) [reply app/views/layouts/application.html.erb:31]: Yes! The perfect placement

File: lib/tasks/product_announcements.rake (hunk 3 above)
Comment (RubenSousaDinis) [reply lib/tasks/product_announcements.rake:9]: Can it work similar to the partnerships task? And receive the path for a JSON file?

Comment (shirleyodeng) [reply lib/tasks/product_announcements.rake]: Done!

**Comment by shirleyodeng** [comment]
@RubenSousaDinis - just realised this is what the footer looks like on my laptop:

Screenshot 2023-04-23 at 21 58 18

This is due to a style (max-height: 80%) on the `InnerContainer` on the `Modal` component in the design system repo. Is it possible to remove this attribute or change so that a className / style prop can be passed to this component?

Screenshot 2023-04-23 at 21 58 42

**Pull request merged**